profilegen/src/res/default.rb

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+module Azure::ARM
+  # Default configuration options for {Azure::ARM.Client}
```

Comment:
Same question of namespace conflict here as well?

Reply:
Same as above